flymake-shellcheck.el

```diff
@@ -30,13 +31,22 @@
 
 ;;; Code:
 
+(defgroup flymake-shellcheck nil
+  "Shellcheck backend for Flymake."
+  :prefix "flymake-shelcheck-"
+  :group 'tools)
+
+(defcustom flymake-shellcheck-path
+  (executable-find "shellcheck")
+  "The path to the `shellcheck' executable."
+
+  :type 'string
+  :group 'flymake-shellcheck)
```

Comment:
The `:group` parameter can be omitted, as Emacs will automatically use the last group defined in the same file. See here: https://www.gnu.org/software/emacs/manual/html_node/elisp/Variable-Definitions.html

Reply:
Thanks for pointing that out.  I've removed it in f4892aa087ec7a6ad00584b2a415626adbfee6d3.